go.mod

```
@@ -1,13 +1,16 @@
 module github.com/smallstep/scep
 
-go 1.16
+go 1.20
```

Would like to keep this version, as the intention is to keep this current version drop-in compatible with `micromdm/scep/v2/scep`. We can start breaking things as soon as we tag an official `v1`, and then go forward with a `v2`.
```suggestion
go 1.16
```